### PR TITLE
Tree table sr

### PR DESCRIPTION
**Pull request by shayan-roshan**
## Description

updated the html so that whenever row title or address is copied, it doesnt copy the screen reader element along with it.
using <div> seemed to fix the problem and it seemed that the issue was only with google chrome

## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/documentation/src/lib/components/tree-table/demos/basic/tree-table-basic.component.html
Comment (cwolf10) [inline libs/documentation/src/lib/components/tree-table/demos/basic/tree-table-basic.component.html:1]: Not sure that we need to update these demos anymore. They are only available by running them locally. Probably should be removed at some point here soon. @davereed, what are your thoughts on removing the old demos?

File: libs/documentation/src/lib/storybook/tree-table/tree-table-basic/tree-table-basic.component.html
Comment (cwolf10) [inline libs/documentation/src/lib/storybook/tree-table/tree-table-basic/tree-table-basic.component.html:1]: "Row Level {level}" announcement is now missing on rows without children.

Also the issue of "Row Level {level}" being copied is still happening. Copy the first two elements in level two to find an example. Or just update the data to give Bill of Rights a child.

**Review by cwolf10 — CHANGES_REQUESTED** [review]

**Comment by cwolf10** [comment]
I don't think this is something for SDS to solve.

First, I don't think it even is an issue. The core functionality of the tree table is to provide collapsable data to both sighted and unsighted users. It does that. This "issue" only shows up when someone copies  multiple lines of data from a tree table. Is this a use case we support? I don't remember that ever being a use case when the tree table was created.

Second, the "issue" is tied to sr-only text being picked up by copy. We get sr-only from USWDS and they have the exact same issue on their site. This can be check by visiting their [Search Component page](https://designsystem.digital.gov/components/search/) where they make use of sr-only. Copying the first searchbar like this:
Screenshot 2023-08-25 at 10 48 14 AM

will yield this output when pasted:
> DEFAULT
Search
Search

If there is an issue, which I don't believe there is, it is with sr-only being available to be selected for copy operations. A solution for that "issue" would have to be provided by USWDS, not SDS.

**Comment by shayan-roshan** [comment]
I agree.
 The primary function of the tree table is being served correctly, and this copy behavior, is largely influenced by external dependencies.

Furthermore, after some testing, I've noticed that this issue seems to be predominantly present on Chrome. This behavior varies based on how the user selects and copies the data, which makes it challenging to cater to every possible user interaction. Given the range of potential user copy methods, it's almost impractical to predict and solve for every scenario.

**Pull request closed without merging**